Jetstream/ModelValue.swift

```diff
@@ -116,7 +116,7 @@ func modelValueIsNillable(type: ModelValueType) -> Bool {
 }
 
 extension String: ModelValue {
-    func equalTo(value: ModelValue) -> Bool { return self == value as String }
+    func equalTo(value: ModelValue) -> Bool { return self == value as! String }
```

Comment:
Hmm, should these values be banged out?  What if you do the following:

```swift
let stringModelValue = "a string" as ModelValue
let intModelValue = 3 as ModelValue
stringModelValue.equalTo(intModelValue) // this will crash as intModelValue as! String will cause a crash
```